packages/batch-submitter/package.json

```diff
@@ -48,7 +48,7 @@
     "prom-client": "^13.1.0"
   },
   "devDependencies": {
-    "@eth-optimism/smock": "^1.1.10",
+    "@eth-optimism/smock": "1.1.10",
```

Comment:
Not strictly necessary to pin smock, but since we aren't planning to release any more versions of smock I think it's best to pin it. I can confirm that 1.1.10 was the version being used.